package.json

```diff
@@ -21,10 +21,7 @@
     "react-dom": "^15.4.2",
     "require-dir": "^0.3.0",
     "gulp-entity-convert": "0.0.2",
-    "gulp-inject-element": "^2.0.2",
     "gulp-insert": "^0.5.0",
-    "gulp-streamify": "^1.0.2",
```

Comment:
@finneganh I believe we use this when downloading the wrapper from the boston.gov API in `_gulp/tasks/download_wrapper.js`. Is there a particular reason you want to remove it here? Thanks!

Reply:
per our conversation in slack "package.json changes are just removing duplicate entries."